addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -670,8 +741,12 @@ var ImageWidget = FileWidget.extend({
     /**
      * @override
      */
-    start: async function () {
-        await this._super(...arguments);
+    start: async function (search, ...args) {
+        const _super = this._super.bind(this);
+        await _super(false, ...args);
+        this.$('.o_we_existing_attachments')
+            .addClass('o_we_search_prompt')
+            .append($(`<h2>${_t("Get the perfect image by searching in our library of copyright free photos and illustrations.")}</h2>`));
```

Comment:
can you double check that really translatable please and confirm. thx

Reply:
Can confirm this does not work: babel treats the entire content of the string literal as a string, even the parts between interpolation sigils. I've extracted the translation outside the template literal and that fixes it, but this will probably get removed anyway and go directly in the XML template once I figure out how to not show any of the default images.